feature/system/logging/remote_syslog/otg_tests/README.md

```diff
@@ -5,39 +5,44 @@
-    *   Configure 3nd IPv6 Syslog remote hosts in _vrf_ with:
-        *   non-standard remote port 
-        *   facility “local1” and severity “debug”
-        *   (TODO when OC model published) complince to RFC5424 (structured)
-        * source address equall to IPv6 address of loopback interface
-    *   Configure 4nd IPv6 Syslog remote hosts in _vrf_ with:
-        * facility “local7” and severity “critical” 
-        * (TODO when OC model published) complince to RFC53164 (BSD/original)
-        * source address equall to IPv6 address of loopback interface
-
-### Test Procedure 
-* Read configuration of all 4 servers, verify it matches intent
-* enable packet capture on OTG port-1
-* disable OTG port-2 so DUT interface(port-2) goes down, what should generate log
-* Observe on OTG capture:
-    *   Syslog packet w/ DstIP of host 1st and 4th and standard dstPort and facility “local7” and “local1” respectively.
-    *   Syslog packet w/ DstIP of host 3rd and non-standard dstPort and facility “local1”
-    *   Note: no packet w/ DstIP of 2nd host is expected.
+*   Configure DUT $VRF-name network-instance and OTG with:
+        *   interface(port-1), interface(port-2) with IPv4 and IPv6 address
+        *   static host routes to syslog servers's addresses pointing OTG interface(port-1) IP
+        *   loopback interface with IPv4 and IPv6 address and netmasks of /32, /64 respectively
+    * Configure  syslog servers DUT
+        *   Configure 1st IPv4 Syslog remote hosts in $VRF-name with:
+            * facility “local7” and severity “debug”
+            * (TODO when OC model published) complince to RFC5424 (structured)
+            * source address equall to IPv4 address of loopback interface
+        *   Configure 2nd IPv4 Syslog remote hosts in $VRF-name with:
+            * facility “local7” and severity “critical” 
+            * (TODO when OC model published) complince to RFC3164 (BSD/original)
+            * source address equall to IPv4 address of loopback interface
+        *   Configure 3nd IPv6 Syslog remote hosts in $VRF-name with:
+            *   non-standard remote port 
+            *   facility “local1” and severity “debug”
+            *   (TODO when OC model published) complince to RFC5424 (structured)
+            * source address equall to IPv6 address of loopback interface
+        *   Configure 4nd IPv6 Syslog remote hosts in $VRF-name with:
+            * facility “local7” and severity “critical” 
+            * (TODO when OC model published) complince to RFC3164 (BSD/original)
+            * source address equall to IPv6 address of loopback interface
+* Test Procedure 
+    * Read configuration of all 4 servers, verify it matches intent
+    * enable packet capture on OTG port-1
+    * disable OTG port-2 so DUT interface(port-2) goes down, what should generate log
+    * Observe on OTG capture:
+        *   Syslog packet w/ DstIP of host 1st and 4th and standard dstPort and facility “local7” and “local1” respectively.
```

Comment:
Why is the 4th expected when it has severity=critical (continuation of https://github.com/openconfig/featureprofiles/pull/1871#discussion_r1263057664)?